src/components/App/App.js

```diff
@@ -34,34 +34,46 @@ class App extends React.Component {
      return polish.brand === newPolish.brand
    })
 
-   const index = this.state.polishes.findIndex(foundPolish => {
-     return foundPolish.brand === newPolish.brand
-   })
-   let copy = [...this.state.polishes]
-
-   if (foundPolish) {
-     const foundColor = foundPolish.colors.find(color => {
-       return color.colour_name === newPolish.colorway
-     })
-     if (!newPolish.hue && foundColor) {
-      newPolish.hue = foundColor.hex_value
+   if (!foundPolish) {
+    this.setState({collection: [...this.state.collection, newPolish]})
+    const formattedToCollection = {
+      id: newPolish.id,
```

Comment:
You're accessing newPolish dot quite a few times, could destructure?

Reply:
Great catch! I will fix that.